### ISSUE TITLE
Crash on joining fresh world. Exit code 9

### ISSUE THREAD

**Issue by almightyfoon** [EnigmaticaModpacks/Enigmatica6#4380]
### Modpack Version

1.0.0

### Describe your issue.

When logging into a new world it gets to the point where my player is in the world but it never fully joins and exits with error code 9.

### Latest Log

https://gist.github.com/almightyfoon/20b5e6c2c27ee95a3a430795495e9943

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by NillerMedDild** [comment]
Thanks for the report. 

I'm afraid there's no crash in your log, could you please try to find the relevant crash report instead of the latest.log? :)

**Comment by almightyfoon** [comment]
It didn't generate a crash, it'll just hang and die and throw a code 9. I've been trying a few other things, I noticed we're past the migration date and I hadn't done it yet so I migrated my account thinking that might be causing it, but now its not even getting to world launch. I'm playing with a few things atm, different java versions mostly. Heres a gist with my latest crash, no crashlog generated so its a hard error somewhere.

https://gist.github.com/almightyfoon/19fb4f46175a04965a3ef3193b43df4c

I'm on pop_os 21, and I made sure everything was up to date as of this morning. Going to try rebooting and see if that makes a difference. My laptops been on for a few days and things tend to get weird with it when that happens.
